libs/extensions/angular/localization/src/amount/amount-service-formatter.ts

```diff
@@ -57,14 +59,19 @@ export function deriveConfiguration(configuration: AmountServiceConfiguration =
   return Object.assign({}, config, configuration);
 }
 
-export type ShowCurrencyCode = '' | 'alwaysShowCurrency' | 'showForeignCurrency';
+export type ShowCurrencyCode =
+  | ''
+  | 'alwaysShowCurrency'
+  | 'showForeignCurrency'
+  | 'showCustomCurrencySymbol';
 export type CurrencyCodePosition = '' | 'prefix' | 'postfix';
 
 export interface AmountServiceConfiguration {
   /**
    * - '' - don't output CurrencyCode
    * - 'alwaysShowCurrency' - always shows CurrencyCode, regardless of presentation currency
    * - 'showForeignCurrency' - only show CurrencyCode if it differs from the presentation currency
+   * - 'showCustomCurrencySymbol' - Shows the currency symbol defined in KirbyExtensionsLocalizationToken.currencyMappings instead of the currency code. Fallback to currencyCode
```

Comment:
This should also be added [here](https://github.com/kirbydesign/designsystem/blob/develop/libs/extensions/angular/localization/src/amount/amount.stories.ts#L58) - there is some unfortunate duplication of these types of docs right now, to give a better experience when reading in storybook.